tests/test_examples.py

```diff
@@ -0,0 +1,25 @@
+import subprocess
+import os
+from pathlib import Path
+
+
+# Currently, these tests will run in the background if no screen is available
+# If you do have a screen available, plots will appear, and you'll need to
+# close them to make the tests pass.
+# TODO: see if we can make these run locally w/o showing matplotlib
+# @patch("matplotlib.pyplot.show")
```

Comment:
Does this work?

Reply:
Not yet...I spent ~15 mins trying a few things and found the commented decorator here: https://stackoverflow.com/questions/51429465/how-to-mock-pyplot-show-in-python-to-prevent-showing-plots

But it doesn't seem to work as I want it to, and I don't have the capacity right now to completely understand how these mocks+decorators work. 

It's a low priority, since this runs smoothly for now...but if you have experience here I'd love to learn.

Reply:
I suspect that this doesn't work because the in-memory patch is not propagated to the subprocesses.

Reply:
But we can deal with this later!